_data/public-speaking.yml

```diff
@@ -1,3 +1,49 @@
+- name: ""
+  by: ""
+  event: ""
+  subTitle: ""
+  date: 
+  summary: >
+    
+  content: ""
+  level: 
+  duration: ""
+  image: ""
+  linkedIn: ""
+
+  - name: ""
```

Comment:
See PR checklist, this will probably crash the site generation
Also, I'm not sure what adding empty entries like this will do (but we can test that:D)